packages/common/src/converter/types.ts

```diff
@@ -30,3 +31,11 @@ export const encodingKeys = {
   METADATA_ENCODING_PROTOBUF_JSON: u8(encodingTypes.METADATA_ENCODING_PROTOBUF_JSON),
   METADATA_ENCODING_PROTOBUF: u8(encodingTypes.METADATA_ENCODING_PROTOBUF),
 } as const;
+
+export const METADATA_MESSAGE_TYPE_KEY = 'messageType';
+
+export interface ProtobufSerializable {
```

Comment:
We could rename this to `ProtobufCodec` because technically this refers to the class and not the serializable instance.